src/main/java/com/exactpro/th2/hand/services/HandBaseService.java

```diff
@@ -536,6 +545,51 @@ private void addGetScreenshot(CSVPrinter printer, GetScreenshot getScreenshot) t
 		printer.println();
 	}
 
+	private void addExecuteJS(CSVPrinter printer, ExecuteJS execJs) throws IOException
+	{
+		printer.print("#action");
+		printer.print("#commands");
+		printer.println();
+
+		printer.print("ExecuteJS");
+		printer.print(execJs.getCommands());
+		printer.println();
+	}
+
+	private void addExecuteJSElement(CSVPrinter printer, ExecuteJSElement execJs) throws IOException
+	{
+		printer.print("#action");
+		printer.print("#commands");
+		printer.print("#wait");
+		printer.print("#locator");
+		printer.print("#matcher");
```

Comment:
One day all these literals will become constants, right? Or this will be replaced with some builder.

Reply:
It should be refactored straight to rh object later without string building